etc/init.d/zfs-mount.in

```diff
@@ -13,10 +13,11 @@
 #
 ### BEGIN INIT INFO
 # Provides:          zfs-mount
-# Required-Start:    $local_fs zfs-import
+# Required-Start:    mountall zfs-import
 # Required-Stop:     $local_fs zfs-import
-# Default-Start:     2 3 4 5
+# Default-Start:     S
```

Comment:
Nit: can you also update the README to reflect that zfs-mount should be started at `S` runlevel?

```
    update-rc.d zfs-import   start 07 S .       stop 07 0 1 6 .
    update-rc.d zfs-load-key start 02 2 3 4 5 . stop 06 0 1 6 .
    update-rc.d zfs-mount    start 02 2 3 4 5 . stop 06 0 1 6 .
    update-rc.d zfs-zed      start 07 2 3 4 5 . stop 08 0 1 6 .
    update-rc.d zfs-share    start 27 2 3 4 5 . stop 05 0 1 6 .
    ```